linode/manager
# refactor: [M3-8056] - Query Key Factory for Kubernetes

## Description 📝

- Query key factory refactor for Kubernetes 🏭 

## Preview 📷

> [!note]
> No UI changes

## How to test 🧪

- Test general create, update, delete operations on Kubernetes clusters and their related node pools 🔍 
- Verify all automated testing related to Kubernetes passes ✅ 

## As an Author I have considered 🤔
